widgets/BGDialog.cpp

```diff
@@ -21,20 +21,21 @@ BGDialog::BGDialog(QWidget *parent) :
 	QDialog(parent, Qt::Tool), field(0)
 {
 	setWindowTitle(tr("Décor"));
+	imageFrame = new QFrame();
+	imageFrame->setMinimumSize(320, 240);
 
-	QScrollArea *scrollArea = new QScrollArea(this);
-	scrollArea->setWidgetResizable(true);
-	scrollArea->setMinimumSize(320, 240);
-
-	QPalette pal = scrollArea->palette();
+	QPalette pal = imageFrame->palette();
 	pal.setColor(QPalette::Active, QPalette::Window, Qt::black);
 	pal.setColor(QPalette::Inactive, QPalette::Window, Qt::black);
 	pal.setColor(QPalette::Disabled, QPalette::Window, .60*Qt::black);
-	scrollArea->setPalette(pal);
+	imageFrame->setPalette(pal);
+
 
 	image = new ApercuBGLabel();
 	image->setAlignment(Qt::AlignCenter);
-	scrollArea->setWidget(image);
+	QVBoxLayout * imageLayout = new QVBoxLayout();

```

Comment:
Please write pointers declaration with one space before and no space after: Foo *bar;
The coding style in MR need changes, but for now I prefer to keep everything consistant.